### PR TITLE
AI and Data: Shorten AI and Data profile summaries in README

### PR DESCRIPTION
**Pull request by bact**
- To make them about the same length as other profiles.
- Proposal from a discussion in AI team meeting [2024-04-24](https://github.com/spdx/meetings/blob/main/ai/2024-04-24.md).
- Revised by comments from 2024-06-12 AI meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,16 +61,9 @@ profiles of the SPDX model.
...
-how the collected data would be used, size of the dataset, information about noise in the
-fields of the dataset and/or noise that affects the entire dataset, information about
-preprocessing before dataset is constructed, sensors used to collect data, known biases
-in the data of the dataset, information about whether the dataset contains sensitive or
-identifiable personal information, anonymization methods used to mask sensitive or identifiable
-personal information, confidentiality levels of data points in the dataset as defined in the
-Traffic Light Protocol, information about dataset update mechanism, and information about
-availability aspects of the dataset (public, restricted, …).
+The Dataset profile describes a dataset's core aspects (type, size, collection
+method) and its preparation for AI use (preprocessing, noise handling, data
```
Comment (kestewart) [inline README.md:65]: Dataset can be used for more than just AI,  so we may want to be careful on this part.

**Review by kestewart — CHANGES_REQUESTED** [review]
Generally this should be fine, but we need to be careful to not write out the expectation that datasets can be used for more than AI.   Example - callibration data used for hardware.

File: README.md (hunk 1 above)
Comment (bact) [reply README.md:65]: Agree. I will adjust it.

**Review by bennetkl — COMMENTED** [review]
Did we decide not to mention about training characteristics for the dataset

**Comment by bact** [comment]
Thanks @bennetkl. I have now updated the summary to include the AI use case and non-AI use case. I put them in the "intended use" part. I think this can address both your concern about not having an AI example (which is one of our major use cases) and @kestewart comments about other use cases.

```text
The Dataset profile describes a dataset's core aspects (type, size, collection
method), access method, preparation (preprocessing, noise handling), intended
use (e.g. hardware calibration, machine learning), and related considerations
(data quality and privacy).
```

What do you think?

**Comment by bact** [comment]
@bennetkl please kindly review the updated summary.

**Comment by goneall** [comment]
@bennetkl @rgopikrishnan91 - Please review the updated text - if OK, we'd like to merge

**Comment by bennetkl** [comment]
Looks good, approved from me

**Review by kestewart — APPROVED** [review]

**Comment by bact** [comment]
Agreed to merge during AI team meeting 2024-07-31 @bennetkl

**Review by bennetkl — APPROVED** [review]
Approved

**Comment by bennetkl** [comment]
merged

    On Wednesday, July 31, 2024 at 05:54:18 p.m. EDT, Arthit Suriyawongkul ***@***.***> wrote:  
 
 


Agreed to merge during AI team meeting 2024-07-31 @bennetkl

**Comment by bennetkl** [comment]
I'm not able to merge, @kestewart , appreciated if you can merge this one.  I approve the edited summaries

**Pull request merged**